deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -72,26 +74,26 @@ metadata:
             "sidecars": [
               {
                 "name": "csi-node-driver-registrar",
-                "repository": "quay.io/k8scsi/csi-node-driver-registrar",
-                "tag": "v1.2.0",
+                "repository": "registry.redhat.io/openshift4/ose-csi-driver-registrar",
+                "tag": "v4.3",
                 "imagePullPolicy": "IfNotPresent"
               },
               {
                 "name": "csi-provisioner",
-                "repository": "quay.io/k8scsi/csi-provisioner",
-                "tag": "v1.3.0",
+                "repository": "registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7",
```

Comment:
did you test all the new sidecars?

Reply:
This branch is used by the tests, I will merge it once they will pass